openoa/plant.py

```diff
@@ -2143,61 +2141,15 @@ def nearest_tower(self, id: str) -> str:
             self.calculate_nearest_neighbor()
         return self.asset.loc[id, "nearest_tower_id"].values[0]
 
+    @classmethod
+    def from_entr(cls, *args, **kwargs):
+        from entr.plantdata import from_entr
```

Comment:
Should this be in a try/except clause and inform users to run `pip install "openoa[entr]"`?

Reply:
Added try/catch here and pointed users to the Entr Alliance's py-entr repo.